.github/workflows/pre-integration.yml

```diff
@@ -16,24 +16,20 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        java: [7, 8, 11, 17, 19-ea]
-        os: [ubuntu-20.04, windows-2022, macos-11]
+        java: [8, 11, 17, 19, 20-ea]
+        os: [ubuntu-22.04, windows-2022, macos-11]
         profile: [default, reflection, asm]
       fail-fast: false
     name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}, ${{ matrix.profile }}
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up JDK ${{ matrix.java }}
       uses: actions/setup-java@v1
```

Comment:
Did you miss a version bump here?
To get this benefit? https://github.com/openjdk/jmh/pull/73#discussion_r920871893

Reply:
Right, thanks for catching this!

Reply:
There's 3.5.1 too, now that it's being updated, probably go to actual latest, so you get better support of newer Java versions.